### PR TITLE
Add Swarm support

### PR DESCRIPTION
**Pull request by tomasro27**

### CODE REVIEW

**Comment by fussybeaver** [comment]
We might need to run this test separately, because we already initialise a swarm for some other tests.

https://github.com/fussybeaver/bollard/blob/9126afa12aec4cc7056375c96330676e47c2d284/resources/dockerfiles/bin/run_integration_tests.sh#L23

**Comment by tomasro27** [comment]
Hello @fussybeaver, how do you suggest we run this test separately?

**Comment by fussybeaver** [comment]
> Hello @fussybeaver, how do you suggest we run this test separately? 

Hey, we probably need to exclude the test case from the default test run using a new test specific feature, and add a section in the CircleCI configuration.

**Comment by tomasro27** [comment]
@fussybeaver thanks for updating the tests. I hadn't had a chance to do it yet.
If everything looks good, we can check in, and let me know if we can do a release with this. It is very much appreciated

**Comment by fussybeaver** [comment]
Good to merge, I'll see if I can find time to release, but I'm fairly snowed under with the day job.

File: .circleci/config.yml [hunk 1]
```diff
@@ -61,6 +61,14 @@ jobs:
       - setup_remote_docker
       - run: docker build -t bollard .
       - run: resources/dockerfiles/bin/run_integration_tests.sh --features time --tests
+  test_swarm:
+    docker:
+      - image: docker:27.3
```
Comment (reneleonhardt) [inline .circleci/config.yml:66]: Can Dependabot be enabled? The latest release is 28.1.

**Pull request merged**